app/controllers/sources.js

```diff
@@ -16,10 +17,54 @@ export default Ember.Controller.extend({
 
     init() {
         this._super(...arguments);
-        this.loadPage();
+        this.loadElasticAggregations();
+    },
+
+    searchUrl: Ember.computed(function() {
```

Comment:
I know this is from the discover page, but there's no reason for this to be a computed property. `loadElasticAggregations` should just call `buildElasticCall` (which probably doesn't even deserve to be a util... but there's no point trying to fix everything right now)